pkgs/applications/audio/pulseaudio-dlna/default.nix

```diff
@@ -0,0 +1,43 @@
+{ fetchFromGitHub, stdenv, pythonPackages, lame, opusTools, faac, flac, sox, vorbisTools }:
+
+pythonPackages.buildPythonApplication rec {
+  pname = "pulseaudio-dlna";
+  version = "2017-11-01";
+
+  src = fetchFromGitHub {
+    owner = "masmu";
+    repo = "pulseaudio-dlna";
+    rev = "4472928dd23f274193f14289f59daec411023ab0";
+    sha256 = "1dfn7036vrq49kxv4an7rayypnm5dlawsf02pfsldw877hzdamqk";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [ dbus-python docopt
+                                                 requests
+                                                 setproctitle
+                                                 protobuf
+                                                 psutil 
+                                                 futures
+                                                 chardet
+                                                 notify2
+                                                 netifaces
+                                                 pyroute2
+                                                 pygobject2
+                                                 lxml
+                                                 zeroconf ]
+                                            ++ [ lame
+                                                 opusTools
+                                                 faac
+                                                 flac
+                                                 sox
+                                                 vorbisTools ];
```

Comment:
Maybe create arguments like `mp3Support ? true, lame` and then use `++ lib.optional mp3Support lame`, so people can disable these optional dependencies.